hm7_1.py

```diff
@@ -0,0 +1,16 @@
+""" Task 1: bulls and cows """
+
+x = input("Введите неповторяющееся четырёхзначное число: ")
```

Comment:
pls add checks:
1) it should be a number
2) only unique numbers are accepted

Reply:
@Stml89 done 👍, please